.github/workflows/issues.yml

```diff
@@ -0,0 +1,77 @@
+name: Automatic Pull Request
+
+on:
+  issues:
+    types: [ opened ]
+
+env:
+  # Swift version 5.6-dev (LLVM 542eceb110d8480, Swift 4323d2fa26a6bf8)
+  # Target: x86_64-unknown-linux-gnu
+  SWIFT_IMAGE: swiftlang/swift@sha256:cc3a796de27ef460b7b4a3dc8f2e7e5aff7fcae989ea49674753a87261ae0448
+
+jobs:
+  add:
+    runs-on: ubuntu-20.04
+    if: contains(github.event.issue.labels.*.name, 'add_package')
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Add Package to JSON
+      run: bash .github/add_package.sh
+      id: package
+      env:
+        GH_BODY: ${{ github.event.issue.body }}
+
+    - name: Validate JSON
+      if: steps.package.outputs.changes == 'true'
+      run: docker run --rm -v "$PWD:/host" -w /host $SWIFT_IMAGE swift validate.swift
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
A pull request will not be created unless the validation step has passed, giving us confidence that we can merge any PR automatically generated by this script.